src/lib/utils/Popper.svelte

```diff
@@ -159,6 +162,10 @@
           for (const [name, handler] of events) element.removeEventListener(name, handler);
         }
       });
+      if (referenceEl) {
+        referenceEl.addEventListener('focusout', hideHandler);
```

Comment:
Should these be `removeEventListener()`?

Reply:
Yes it should. Thanks for a catch.